grails-app/controllers/au/org/ala/biocollect/merit/ProxyController.groovy

```diff
@@ -35,11 +35,11 @@ class ProxyController {
     }
 
     def features(){
-        render webService.get("${grailsApplication.config.spatial.layersUrl}/objects/${params.layerId}", false)
+        render webService.get("${grailsApplication.config.spatial.layersUrl}/objects/${params.layerId}")
```

Comment:
looks like this will call `webService.get()` with `includeUserId = true` but it should be false?

Reply:
Reverted in 14fc0d1db14c53cf1143e63d87ffe42f8fd13313